<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanalistadoble</class>
 <widget class="QDialog" name="ventanalistadoble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>280</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="InsertarNumero">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Posición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ObtenerPosBTN">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>obtener por posición</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posición Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarInicioBTN">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar al inicio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosEditarObtener">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarPOSBTN">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar en un posición</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosicionInsertar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="EditarPosBTN">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>editar por posición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarInicioBTN">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar al inicio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumeroEditar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="PosEliminar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertarFinalBTN">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>insertar al final</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar número</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarPOSBTN">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar por posición</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion eliminar</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="EliminarFinalBTN">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>eliminar al final</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista Doble</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ventanalistadoble</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ventanalistadoble</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
